week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -41,19 +41,25 @@ console.log(twoDimensionalArray[1][2]);
 
 // Examples accessing to array element through array methods
 // Example 1
-let moreThanTwoArrays = twoDimensionalArray.filter(embeddedArray => embeddedArray.length > 2);
+let moreThanTwoArrays = twoDimensionalArray.filter(
+  (embeddedArray) => embeddedArray.length > 2
+);
 console.log(moreThanTwoArrays);
 // This filter only keeps embededd arrays that have more than 3 elements
 
 // Example 2
-let arrayLengths = twoDimensionalArray.map(embeddedArray => embeddedArray.length);
+let arrayLengths = twoDimensionalArray.map(
+  (embeddedArray) => embeddedArray.length
+);
 console.log(arrayLengths);
 // This gives back how many elements of each array have
 
 /* + + + + + + + */
 /* Now try the exercise */
 
-function getEligibleStudents() {
+function getEligibleStudents(ages) {
+  let bigger = ages.filter((newArray) => newArray[1] >= 8).map((x) => x[0]);
+  return bigger;
 }
```

Comment:
This function is correct the only advice I can give you is to name your parameters your variables with what you are doing (it should have meaning)